app/models/binary/DataSet.scala

```diff
@@ -423,10 +422,11 @@ case class DataSet(
 object DataSet extends FoxImplicits {
   implicit val dataSetFormat = Json.format[DataSet]
 
-  def dataSetPublicWrites(d: DataSet, user: Option[UserSQL])(implicit ctx: DBAccessContext): Fox[JsObject] =
+  def dataSetPublicWrites(d: DataSet, user: Option[UserSQL]): Fox[JsObject] = {
+    implicit val ctx = GlobalAccessContext
```

Comment:
Ss this just refactoring or is the parameter used somewhere else, too?

Reply:
not just refactoring, this gives GlobalAccessContext rights to the functions below, instead of just the db rights of the requesting user.
This failed before on public datasets requested by someone who could not see the oganization.